app/controllers/LiabilityCalculationController.scala

```diff
@@ -28,13 +30,14 @@ import scala.concurrent.ExecutionContext
 class LiabilityCalculationController @Inject()(liabilityCalculationService: LiabilityCalculationService,
                                                cc: ControllerComponents,
                                                authorisedAction: AuthorisedAction)
-                                              (implicit ec: ExecutionContext) extends BackendController(cc) {
+                                              (implicit ec: ExecutionContext) extends BackendController(cc) with Logging {
 
   def calculateLiability(nino: String, taxYear: String, crystallise: Boolean): Action[AnyContent] = authorisedAction.async { implicit user =>
     liabilityCalculationService.calculateLiability(nino, taxYear, crystallise).map {
       case Right(value) => Ok(Json.toJson(value))
-      case Left(error) => Status(error.status)(error.toJson)
+      case Left(error) =>
+        logger.error(s"[LiabilityCalculationController][calculateLiability] - Error Response: $error")
```

Comment:
not sure if we have to log nino, but we would need to log taxYear to identify exact log entry we need to find in the log